ReactiveCocoa/Swift/Action.swift

```diff
@@ -134,7 +134,7 @@ public final class Action<Input, Output, Error: ErrorType> {
 public final class CocoaAction: NSObject {
 	/// The selector that a caller should invoke upon a CocoaAction in order to
 	/// execute it.
-	public static let selector: Selector = "execute:"
+	public static let selector: Selector = #selector(CocoaAction.execute(_:))
```

Comment:
The current release does build with Swift 2.2, so I'm :+1: to be 2.2+ only in the next release.

I do think it should be 5.0 if we're dropping Swift 2.1 compatibility, but I'll defer to you.